inspirehep/utils/citations.py

```diff
@@ -42,8 +44,7 @@ def citations(self):
 
         out = []
         row = []
-        recid = self.record['control_number']
-        es_query = Query('refersto:' + str(recid)).search()
+        es_query = Query('refersto:' + str(self.recid)).search()
```

Comment:
I think that makes sense. Nowadays we show 5 https://inspirehep.net/record/724189/citations and I don't think there have been requests to show more.

I think the code does not need to be modified much, we can limit the query to 10 results, each row will be outputted and on the client side we will display them (without datatables).

We also need to get the `len()` of the `refersto:` query to show the total number of citations.

I can do the modifications on the client side after we merge this PR.

@tomaszgy do you think making these changes will alter the way we do the caching?